test/testCompileTests.fs

```diff
@@ -0,0 +1,91 @@
+module ts2fable.testCompileTests
```

Comment:
Why use a test to write the files ?

Why not call `ts2fable` cli directly. I don't see the benefit of using `Mocha` to the write the bindings output.

It could easily be done by adding a FAKE target.

Reply:
I didn't think two fable ports are good idea,
And i want run all tests (including test-compile tests) without interrupting webpack work
I also add a cli test using `fake`
  ```fsharp
node <| sprintf "%s node_modules/typescript/lib/typescript.d.ts test-compile/TypeScript.fs" (distDir</>"ts2fable.js")
```
I think this is enough

Reply:
Whatever
I think this is a dogfood repositoy.
tring different things is good for us

Reply:
Why do you speak about running 2 Fable ports/process ?

When users do `ts2fable` it don't run a Fable process. And also, you can run the tests and if the tests pass then generate the bindings. I don't see why generating a binding is a test, it's just writing text to a file but don't execute any code against the generating bindings.

What are you testing here ? That `true = true` ?

```fs
    it "node" <| fun _ ->
        writeFile ["node_modules/@types/node/index.d.ts"] "test-compile/Node.fs"
        equal true true     
```

Reply:
https://github.com/fable-compiler/ts2fable/issues/155
When you changed source code,
The ts2fable.js is not auto-updated
But the tests files are auto-updated

Reply:
This code looks strange 
But in thisway we can prefix with `only` and `onlys` to debug single test
```fsharp
    it "node" <| fun _ ->
        writeFile ["node_modules/@types/node/index.d.ts"] "test-compile/Node.fs"
        equal true true  
```

Reply:
Ok, so this is to work around a problem then.

I will take time when I finish 1-2 of my current project to review the repo and propose a change.

Reply:
Please tell me if my opinion is wrong 
Thanks

Reply:
I just don't think generating a binding is a test by itself. :)

No worries, I will propose a PR when I have the time. This will be easier to explain my point and confirm/invalidate it :)